### PR TITLE
New design on module positions page

### LINKED ISSUE #11102: Drag & drop on migrated positions page doesn't looks like the mockup
type : tech-issue

**Describe the bug**

@TristanLDD made mockups for drag & drop in BO
We need to include it on migrated positions page

**Screenshots**
Currently it looks like this:

capture d ecran_489

Mockup https://zpl.io/aMvjjj7

capture d ecran_490

**Additionnal information**
PrestaShop version: N/A
PHP version: N/A

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | New design on position page following mock-up.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #11102
| How to test?  | Before<br>capture d ecran_489<br>After<br>image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12646)

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/pages/_modules.scss [hunk 1]
```diff
@@ -611,20 +683,34 @@ form#importDropzone {
...
+    }
+  }
+
+  .module-column-infos {
+    width: 200px;
+    padding: 0 4px;
+  }
+
+  .module-column-description {
+    width: 679px;
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/scss/pages/_modules.scss:761]: Weird to have this kind of width, is it ok with different size than yours, like responsive?

File: admin-dev/themes/new-theme/scss/pages/_modules.scss [hunk 2]
```diff
@@ -569,22 +566,99 @@ form#importDropzone {
     display: table;
     width: 100%;
     padding: 5px 0;
-    margin-bottom: -1px;
-    border: solid 1px #9ed0ec;
+    border-bottom: solid 1px #bbcdd2;
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/scss/pages/_modules.scss]: use variable for color please. $gray-light

File: admin-dev/themes/new-theme/scss/pages/_modules.scss [hunk 3]
```diff
@@ -569,22 +566,99 @@ form#importDropzone {
...
     background: #fff;
 
+    &:hover {
+      background-color: #fafbfc;
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/scss/pages/_modules.scss]: $gray-soft

File: admin-dev/themes/new-theme/scss/pages/_modules.scss [hunk 4]
```diff
@@ -569,22 +566,99 @@ form#importDropzone {
...
+
+      .drag_indicator {
+        color: #b3c7cd;
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/scss/pages/_modules.scss]: add new variable :D

File: admin-dev/themes/new-theme/scss/pages/_modules.scss [hunk 5]
```diff
@@ -569,22 +566,99 @@ form#importDropzone {
...
+
+    &:focus {
+      outline: none;
+    }
+
+    &:active,
+    &:visited {
+      border-radius: 3px;
+      box-shadow: 3px 3px 8px 0 rgba(0, 0, 0, 0.2);
+      background-color: #6c868e;
```
Comment (PierreRambaud) [inline admin-dev/themes/new-theme/scss/pages/_modules.scss]: $medium-gray

File: admin-dev/themes/new-theme/scss/pages/_modules.scss (hunk 2 above)
Comment (Progi1984) [reply admin-dev/themes/new-theme/scss/pages/_modules.scss]: :ok_hand:

File: admin-dev/themes/new-theme/scss/pages/_modules.scss (hunk 3 above)
Comment (Progi1984) [reply admin-dev/themes/new-theme/scss/pages/_modules.scss]: :ok_hand:

File: admin-dev/themes/new-theme/scss/pages/_modules.scss (hunk 5 above)
Comment (Progi1984) [reply admin-dev/themes/new-theme/scss/pages/_modules.scss]: :ok_hand:

**Comment by TristanLDD** [comment]
Hi, there are a few design modifications to do on this PR to make the design consistant with other migrated listings :    

Group 39

1 / 2 - Use material icon check box
3 - Use same icon size as in category page
4 - Use only the number, no input
5 - Use same icon size as in category page
6 - Use same hovers as in category page

Also, when you click on a checkbox in order to select it, the entire row changes color as if it was about to be drag and dropped in another position. The drag and drop behavior shouldn't apply to the checkbox. 


Thanks a lot !

**Comment by TristanLDD** [comment]
Hi @Progi1984 

Here are the final design requests for this page: 

Positions2

1. 8px of spacing between checkbox and text 
2. Add tooltip « Edit » on hover (see category page) 
3. Add color change on hover (#25B9D7)
4. Check box is not visible enough so we need to change the style of this componant

Thanks a lot!

**Comment by Progi1984** [comment]
@TristanLDD As asked : 
image

File: admin-dev/themes/new-theme/scss/pages/_modules.scss (hunk 1 above)
Comment (PierreRambaud) [reply admin-dev/themes/new-theme/scss/pages/_modules.scss:761]: up!

Comment (Progi1984) [reply admin-dev/themes/new-theme/scss/pages/_modules.scss:761]: Yeah this div block only display in medium screens. See the positions.html.twig (l.132).

**Review by PierreRambaud — DISMISSED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**